metrics_gocdb.py

```diff
@@ -172,6 +172,72 @@ def get_queries():
     result = elastic.search(index="logstash-" + date, body=params_dict)
     return result["hits"]["total"]
 
+
+def get_unique_ips_over_28_days():
+    """
+    Get the number of unique IP accessing GOCDB over the last 28 days.
+
+    Only accesses via the load balancers will be counted.
+    """
+    # Use a set to leverage the fact duplicates are handled by the data
+    # structure.
+    unique_ips = set()
+    # For the last 28 days (not counting today), query the unique IPs per day.
+    date = datetime.now()
+    for _ in range(0, 28):
+        date = date - timedelta(days=1)
```

Comment:
I agree with @tofu-rocketry and this is still hard to follow, why not do something like:

```python
today = datetime.now()
for i in range(0, 28):
    date = today - timedelta(days=i)
    …
```

Reply:
Can I not do as @jrha sugest @tofu-rocketry? I'd have to fuss with the loop start a bit to have it loop over the same days `(today - timedelta(days=0))` would equal `today`....

I could also move defining `ELASTICSEARCH` and the `params_dict` out of the loop (as they are constants) to make the loop smaller and easier to follow?

Reply:
Ah, I see, you'd always calculate `date` in  relation to today. Yes, that seems better.

Reply:
change made